tools/BuildAssets/psModules/CodeGenerationModules/AutoRestCodeGenerationModule/AutoRestCodeGenerationModule.psm1

```diff
@@ -130,7 +131,9 @@ param(
     [Parameter(Mandatory = $false)]
     [string] $ConfigFileTag,
     [Parameter(Mandatory = $true, HelpMessage ="Please provide a version for the AutoRest release")]
-    [string] $AutoRestVersion
+    [string] $AutoRestVersion,
+    [Parameter(Mandatory = $false)]
+    [string] $ApiType
```

Comment:
@dsgouda let's make this parameter as "SdkGenerationType"
Also add [ValidateSet] and possible value should be "SingleApi", "MultiApi"

Reply:
I needed that feedback :)